federation-js/src/composition/validate/preNormalization/__tests__/rootFieldUsed.test.ts

```diff
@@ -95,6 +97,12 @@ describe('rootFieldUsed', () => {
       Array [
         Object {
           "code": "ROOT_QUERY_USED",
+          "locations": Array [
```

Comment:
Should this location point to the name `Query` itself?

Reply:
The columns for the types start at `type` for all of these I believe.

Reply:
(except that weird graphql error that I can't figure out)